src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
```

Comment:
Avoid hardcoding width values. Your container alignment style (`flex`, `grid`) should be enough to create responsive nicely working container with cards, which would be much easier to maintain